src/CSLOLUtils.cpp

```diff
@@ -118,10 +118,84 @@ bool CSLOLUtils::isUnnecessaryAdmin() {
     }
     return result;
 }
-#else
-// TODO: macos implementation
-QString CSLOLUtils::detectGamePath() { return ""; }
+#elif __APPLE__
```

Comment:
```suggestion
#elif defined(__APPLE__)
```
This needs to check for define

Reply:
:+1: